homeworks/oleksandr.khodakivskyi_Moo0oony/5-logic/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Basic JS</title>
+        
+        <link rel="stylesheet" href="./styles/style.css">
+    </head>
+    <body>
+        <header class="header">
```

Comment:
Proly, you can remove this heade, cause the links don't lead anywhere
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/aa63fa46-630e-45db-bf00-a843d343abb4)

Reply:
Or, you can remove whole `5-logic` folder, as you like to keep calculator as a page of your GamesProject